app/views/shared/_invite_someone_who_can.html.erb

```diff
@@ -1,19 +1,10 @@
 
-<p class="govuk-body">Invite someone in your school or college who:</p>
-<ul class="govuk-list govuk-list--bullet govuk-!-margin-bottom-6">
-  <li>understands what devices are needed and who will get them</li>
-  <li>can configure devices or give technical information</li>
-</ul>
-
-<h2 class="govuk-heading-s">
-  Get consent for us to share their information
-</h2>
 <p class="govuk-body">
-  When you invite someone, their name, email address and <span class="app-no-wrap">phone number</span> will be shared with TechSource. Before you invite them:
+  When you invite someone, their name, email address and <span class="app-no-wrap">phone number</span> will be shared with Computacenter (UK) Limited, the company that provides the Support Portal. Before you invite them:
```

Comment:
This looks like a permanent change

Reply:
True, I'll make this change in a seperate branch

Reply:
True, I'll make this change separately